### ISSUE TITLE
v0.4.0 window unusable and crash

### ISSUE THREAD

**Issue by fleeting-aardvark** [Goz3rr/SatisfactorySaveEditor#74]
When starting up v0.4.0, the window frame shows up but there is no contents {no menu or anything else}.  When attempting to move the window, the program crashes.  Using Windows 8.1.

The previous version 0.3.2 opens just fine.

The following is the problem that the OS shows:
```
Problem signature:
  Problem Event Name:	CLR20r3
  Problem Signature 01:	SatisfactorySaveEditor.exe
  Problem Signature 02:	1.0.0.0
  Problem Signature 03:	fe805c50
  Problem Signature 04:	PresentationCore
  Problem Signature 05:	4.7.3324.0
  Problem Signature 06:	5c09c26d
  Problem Signature 07:	577f
  Problem Signature 08:	13
  Problem Signature 09:	System.OutOfMemoryException
  OS Version:	6.3.9600.2.0.0.256.48
  Locale ID:	1033
  Additional Information 1:	419f
  Additional Information 2:	419f1881761ef763d564a3e0f5fc96fc
  Additional Information 3:	50a4
  Additional Information 4:	50a4c960bdd6950ced3ba63fbac93be8
```

Edit: Further testing shows that the newer version does seem to work under Windows 10, but *not* Windows 8.1.

**Comment by Goz3rr** [comment]
That's definitely very strange, and your .NET framework seems reasonably up to date.

For starters could you try running the program from a cmd window, that should hopefully give some more meaningful logging output.
The easiest way to do this is go to the folder where SatisfactorySaveEditor.exe is, click on the address bar and type in `cmd.exe`, then in the command prompt enter `SatisfactorySaveEditor.exe` and press enter.

**Comment by fleeting-aardvark** [comment]
It does the same thing.*
By same thing, I mean that running the program from the command line does the same as starting it from a shortcut.  Attached is the window that shows up, but unless I try to move the window is only when the error occurs, otherwise nothing else happens with the program.

![Satisfactory_Editor_Bug](https://user-images.githubusercontent.com/42421259/56088371-015ae800-5e45-11e9-961f-aadcc108ed8c.png)

**Comment by fleeting-aardvark** [comment]
Additional notes, I did restart my computer just in case.  I also made sure the OS was up to date {there was a roll up update for .Net}.  Those did not help.

I am not sure what else to try.

**Comment by Goz3rr** [comment]
Yeah I wasn't expecting it to start working then, but did you get any text output in the console window?

**Comment by Goz3rr** [comment]
Could you give this version a try? It should create a log.txt next to the .exe file hopefully giving some more insight.

[SatisfactorySaveEditor-logdebug.zip](https://github.com/Goz3rr/SatisfactorySaveEditor/files/3077134/SatisfactorySaveEditor-logdebug.zip)

**Comment by fleeting-aardvark** [comment]
My apologies, I did not mean to imply that it would start working. No I did not get any output from the command line, as the program opened the window correctly.  In looking at the log from the above posted version, it seems to have insufficient memory to run correctly.  My system has 32gb with at least 28gb free.  Plenty enough to run the program.  So whatever changed between v0.3.2 and v0.4.0 is where the problem started.

[log.txt](https://github.com/Goz3rr/SatisfactorySaveEditor/files/3077603/log.txt)
